examples/basic_tracer/docker/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "2"
+services:
+
+  # Collector
+  collector:
+    image: omnition/opentelemetry-collector-contrib:latest
+    command: ["--config=/conf/collector-config.yaml", "--log-level=DEBUG"]
```

Comment:
Can we configure the collector to log when it receives spans? I'm surprised to see it doesn't do this at DEBUG level when running the example code.

Reply:
Added logging exporter in Collector config